python/equistore-operations/equistore/operations/reduce_over_samples.py

```diff
@@ -149,7 +149,6 @@ def _reduce_over_samples_block(
 
     if reduction == "mean" or reduction == "std" or reduction == "var":
         bincount = _dispatch.bincount(index)
-        bincount = _dispatch.array_like_data(values_result, bincount)
```

Comment:
These are leftovers from the old Labels class where the values have to be numpy arrays.